frontend/integration-tests/tests/monitoring.scenario.ts

```diff
@@ -245,7 +245,10 @@ describe('Alertmanager: Configuration', () => {
     await firstElementByTestID('input-repeat-interval').sendKeys('24h');
 
     await monitoringView.saveButton.click();
-    await crudView.isLoaded();
+    browser.wait(
+      until.textToBePresentInElement(firstElementByTestID('repeat_interval_value'), '24h'),
+      5000,
```

Comment:
I'd leave off the custom timeout... Just use the default

Reply:
ok, removed the 5000